security/pkg/nodeagent/cache/secretcache.go

```diff
@@ -87,6 +93,9 @@ const (
 
 	// DefaultRootCertFilePath is the well-known path for an existing root certificate file
 	DefaultRootCertFilePath = "./etc/certs/root-cert.pem"
+
+	// CSR empty token, will be applied when the CSR request doesn't contain a token
+	EmptyToken = ""
```

Comment:
Is there any reason for this constant ? Any case where the empty token will not be "" ?

Reply:
Will remove it. When CSR request not needs the token it will use empty as input